pyproject.toml

```
@@ -1,16 +1,73 @@
 [build-system]
-requires = ["flit_core >=2,<4"]
-build-backend = "flit_core.buildapi"
-
-[tool.flit.metadata]
-module = "pytest_celery"
-dist-name = "pytest-celery"
-author = "Thomas Grainger"
-author-email = "pytest-celery@graingert.co.uk"
-home-page = "https://github.com/graingert/pytest-celery"
-license = "BSD"
+build-backend = "poetry.core.masonry.api"
+requires = ["poetry-core>=1.0.0"]
+
+[tool.black]
+exclude = '''
+/(
+    \.git
+  | \.hg
+  | \.mypy_cache
+  | \.tox
+  | \.venv
+  | _build
+  | buck-out
+  | build
+  | dist
+  | \.idea
+  | \.github
+)/
+'''
+include = '\.pyi?$'
+line-length = 120
+
+[tool.isort]
+line_length = 120
+profile = "black"
+
+[tool.poetry]
+authors = [
+    "Thomas Grainger <pytest-celery@graingert.co.uk>",
+    "Omer Katz <omer.katz@omerkatz.com>",
+    "Naomi Elstien <naomi.els@omerkatz.com>"
+]
 classifiers = ["License :: OSI Approved :: BSD License"]
-requires = ["celery >= 4.4.0"]
+description = "Pytest plugin for Celery"
+homepage = "https://github.com/graingert/pytest-celery"
```

celery/pytest-celery?

---

If this is a simple change, you can use a suggestion.

```suggestion
homepage = "https://github.com/celery/pytest-celery"
```

---

I actually don't  know how to use this feature though I like it :D

---

See https://docs.github.com/en/pull-requests/collaborating-with-pull-requests/reviewing-changes-in-pull-requests/commenting-on-a-pull-request#adding-line-comments-to-a-pull-request